### ISSUE TITLE
giphy integration

### ISSUE THREAD

**Issue by kentcdodds** [sindresorhus/refined-github#72]
I don't use refined-github yet, but I think that if there were giphy integration it might finally push me over the edge.

I'm thinking that putting something by textareas that I can click on and search for a giphy gif that I could insert into my comment, that'd be terrific. Something like what twitter has:

<img width="517" alt="screen shot 2016-03-10 at 7 39 51 am" src="https://cloud.githubusercontent.com/assets/1500684/13674681/4e7d5820-e693-11e5-8230-d860127ff891.png">

**Comment by DrewML** [comment]
I really liked this idea, and wanted it for personal use, so [I ended up throwing it together](https://github.com/DrewML/GifHub).

I did *not*, however, add it to this project. The only reason for not contributing directly was that there are a few features of this project I don't need personally, and I understand (and am plenty fine with) the decision to not allow toggling features in Refined Github. I do appreciate all of @sindresorhus's hard work on it, though :).
